index.js

```diff
@@ -31,10 +35,11 @@ module.exports = function packager (opts, cb) {
     default: return cb(new Error('Unsupported platform. Must be either darwin, linux, or win32'))
   }
 
-  // Ignore this and related modules by default
-  var defaultIgnores = ['/node_modules/electron-prebuilt($|/)', '/node_modules/electron-packager($|/)', '/\.git($|/)']
-  if (opts.ignore && !Array.isArray(opts.ignore)) opts.ignore = [opts.ignore]
-  opts.ignore = (opts.ignore) ? opts.ignore.concat(defaultIgnores) : defaultIgnores
+  // Only include files specified
+  var defaultFiles = ['package.json']
+  opts.files = (defaultFiles + config.files).map(function (item) {
+    path.join(opts.dir, item)
```

Comment:
also just to clarify, is the idea to change the CLI API from `electron-packager <sourcedir>` to e.g. `electron-packager --files=foo/bar.js --files=foo/baz.js` where you specify each individual file on the CLI?